mrp_repair_fee/models/mrp_repair.py

```diff
@@ -46,24 +46,27 @@ def _catch_default_to_invoice(self):
     def _onchange_user_id(self):
         employee_obj = self.env['hr.employee']
         result = {}
```

Comment:
Yo habría metido un:
if not self.user_id: return {}
Y mantenías el resto del código al mismo nivel